tests/framework/matcher/status.go

```diff
@@ -82,3 +83,27 @@ func HaveStateChangeRequests() gomegatypes.GomegaMatcher {
 		}),
 	}))
 }
+
+func InterfaceByName(interfaceName string) func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstanceNetworkInterface {
+	return func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstanceNetworkInterface {
+		for _, networkInterface := range vmi.Status.Interfaces {
+			if networkInterface.Name == interfaceName {
+				return &networkInterface
+			}
+		}
+		return nil
+	}
+}
+
+func InterfaceIPs(networkInterface *v1.VirtualMachineInstanceNetworkInterface) []string {
+	if networkInterface == nil {
+		return nil
+	}
+	return networkInterface.IPs
+}
+
+func MatchIPsAtInterfaceByName(interfaceName string, ipsMatcher gomegatypes.GomegaMatcher) gomegatypes.GomegaMatcher {
+	return WithTransform(InterfaceByName(interfaceName), SatisfyAll(
+		Not(BeNil()), WithTransform(InterfaceIPs, ipsMatcher)))
```

Comment:
I'm OK with the satisfy all clause, but this became harder to reason about; please provide better indentation to make this more straightforward to the reader.

Something like
```suggestion
	return WithTransform(
	    InterfaceByName(interfaceName),
	    SatisfyAll(
		Not(BeNil()), 
		WithTransform(InterfaceIPs, ipsMatcher)))
```